test/format/format.sh

```diff
@@ -0,0 +1,301 @@
+	echo "$0: format program \"$format_binary\" not found"
+	exit 1
+}
+[[ $abort_test -ne 0 ]] && [[ ! -x "$wt_binary" ]] && {
+	echo "$0: wt program \"$wt_binary\" not found"
+	exit 1
+}
+[[ -z "$config" ]] && config="$build/CONFIG.stress"
+[[ -f "$config" ]] || {
+	echo "$0: configuration file \"$config\" not found"
+	exit 1
+}
+[[ -d "$home" ]] || {
+	echo "$0: run directory \"$home\" not found"
+	exit 1
+}
+
+[[ $verbose -ne 0 ]] &&
+    echo "$0 [-c $config] [-h $home] [-j $jobs] [-n $runs] [-t $minutes] $format_args"
+
+failure=0
+success=0
+# Resolve/cleanup completed jobs.
+resolve()
+{
+	for dir in $home/RUNDIR.[0-9]*; do
+		# Skip log files and failures we've already reported.
+		[[ ! -d $dir ]] || [[ -f "$dir/reported" ]] && continue
+		log="$dir.log"
+
+		# Discard successful runs.
+		grep 'successful run completed' $log > /dev/null && {
+			rm -rf $dir $log
+			success=$(($success + 1))
+			[[ $verbose -ne 0 ]] && echo "$0: job in $dir successfully completed"
+			continue
+		}
+
+		# Test recovery on runs configured for random abort. */
+		grep 'aborting to test recovery' $log > /dev/null && {
+			cp -pr $dir $dir.RECOVER
+
+			(echo
+			 echo "$0: running recovery after abort test"
+			 echo "$0: original directory copied into $dir.RECOVER"
+			 echo) >> $log
+
+			# Everything is a table unless explicitly a file.
+			uri="table:wt"
+			grep 'data_source=file' $dir/CONFIG > /dev/null && uri="file:wt"
+						
+			# Use the wt utility to recover & verify the object.
+			if  $($wt_binary -R -h $dir verify $uri >> $log 2>&1); then
+				rm -rf $dir $dir.RECOVER $log
+				success=$(($success + 1))
+				[[ $verbose -ne 0 ]] &&
+				    echo "$0: job in $dir successfully completed"
+			else
+				echo "$0: failure status reported" > $dir/reported
+				failure=$(($failure + 1))
+				echo "$0: job in $dir failed abort/recovery testing"
+			fi
+			continue
+		}
+
+		# Discard runs where the timer went off.
+		grep 'caught signal' $log > /dev/null && {
+			rm -rf $dir $log
+			[[ $verbose -ne 0 ]] && echo "$0: job in $dir aborted"
+			continue
+		}
+
+		# Report failures.
+		grep 'RUN PARAMETERS' $log > /dev/null && {
+			echo "$0: failure status reported" > $dir/reported
+			failure=$(($failure + 1))
+			echo "$0: job in $dir failed"
+		}
+	done
+	return 0
+}
+
+# Start a single job.
+count_runs=0
+run_format()
+{
+	count_runs=$(($count_runs + 1))
+	dir="$home/RUNDIR.$count_runs"
+	log="$dir.log"
+
+	if [[ $smoke_test -ne 0 ]]; then
+		args=${smoke_list[$smoke_next]}
+		smoke_next=$(($smoke_next + 1))
+		echo "$0: starting smoke-test job in $dir"
+	else
+		args=$format_args
+
+		# If abort/recovery testing is configured, do it 5% of the time.
+		[[ $abort_test -ne 0 ]] && [[ $(($count_runs % 20)) -eq 0 ]] && args="$args abort=1"
+
+		echo "$0: starting job in $dir"
+	fi
+	[[ $verbose -ne 0 ]] &&
+	    echo "$0: running $format_binary -c "$config" -h "$dir" -1 $args quiet=1"
+	$format_binary -c "$config" -h "$dir" -1 $args quiet=1 > $log 2>&1 &
```

Comment:
Prefer to put the command into a variable, so that we don't need to remember updating 2 places next time there's a need to tweak the command.

Reply:
Fixed in 4261b9a93.